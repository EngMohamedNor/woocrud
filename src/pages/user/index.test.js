import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';


import UserProfile from './index';

configure({adapter: new Adapter()});

it('should render correctly with no props', () => {
    const component = shallow(<UserProfile/>);
    expect(component).toMatchSnapshot();
});
