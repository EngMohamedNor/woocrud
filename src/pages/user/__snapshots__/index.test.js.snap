// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile should render correctly in "debug" mode 1`] = `
<div
  id="page-404"
>
  <div
    className="not-found-inner"
  >
    <p>
      Profile Page
    </p>
    <WithStyles(ForwardRef(Button))
      onClick={[Function]}
      variant="outlined"
    >
      Page D'acceuil
    </WithStyles(ForwardRef(Button))>
  </div>
</div>
`;

exports[`should render correctly with no props 1`] = `
<div
  id="page-404"
>
  <div
    className="not-found-inner"
  >
    <p>
      Profile Page
    </p>
    <WithStyles(ForwardRef(Button))
      onClick={[Function]}
      variant="outlined"
    >
      Page D'acceuil
    </WithStyles(ForwardRef(Button))>
  </div>
</div>
`;
